Clarinet.toml

```diff
@@ -302,6 +302,12 @@ depends_on = [
   "trait-multisig-vote"
 ]
 
+[contracts.agp019]
```

Comment:
Did this sneak into the PR? Regardless, looks good to me and perhaps a good principle to adopt in general. (I.e. use contract-caller unless the implementation demands tx-sender.)